### PR TITLE
BTFS-1649

### PR DESCRIPTION
**Pull request by steveyeom**
Added functions for convert for go-btfs-api

### CODE REVIEW

**Comment by taiyangc** [comment]
No code?

File: utils/convert.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		if err != nil {
+			return nil, err
+		}
+		return by, nil
+	default:
+		return nil, fmt.Errorf(`unexpected encoding [%d], expected 1(Text) or 2(Base64)`, encoding)
+	}
+}
+
+func BytesToString(data []byte, encoding int) (string, error) {
```
Comment (taiyangc) [inline utils/convert.go]: add one unit test for each function please.

**Comment by steveyeom** [comment]
Added test

File: utils/convert.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	"encoding/base64"
+	"fmt"
+)
+
+const (
+	Text = iota + 1
+	Base64
+)
+
+func StringToBytes(str string, encoding int) ([]byte, error) {
```
Comment (wxue) [inline utils/convert.go]: Shouldn't this belong to go-common repo?
similar to https://github.com/TRON-US/go-common/blob/master/operator/string.go

Comment (taiyangc) [reply utils/convert.go]: It is a little btfs-offline-related by defining custom encoding/decoding schema

Comment (taiyangc) [reply utils/convert.go]: But yeah, can be general.

**Pull request merged**